packages/skeleton/src/lib/components/Accordion/AccordionItem.svelte

```diff
@@ -64,6 +68,15 @@
 	export let regionPanel: CssClasses = getContext('regionPanel');
 	/** Provide arbitrary classes caret icon region. */
 	export let regionCaret: CssClasses = getContext('regionCaret');
+	// ---
+	/** DEPRECATED: use transition or transitionIn, transitionOut instead. */
+	export let duration: number = getContext('duration');
+	// Silence warning about unused props:
+	const deprecated = [duration];
+
+	// Local
+	const { transition: trIn, params: trInParams } = transitionIn;
+	const { transition: trOut, params: trOutParams } = transitionOut;
```

Comment:
Would be another breaking change for those that implement `duration` for this component. 
```suggestion
	/** DEPRECATED: use transition or transitionIn, transitionOut instead. */
	export let duration: number = getContext('duration');

	// Local
	const { transition: trIn, params: trInParams } = transitionIn;
	const { transition: trOut, params: trOutParams } = transitionOut;
```

...(continued below in the next comment)